packages/summary-list/src/SummaryList.tsx

```diff
@@ -10,7 +10,13 @@ export type SummaryListItemTermProps = PropsWithOnlyChildren;
 /** SummaryListItemTerm should be nested as the first child of SummaryListItem */
 export function SummaryListItemTerm({ children }: SummaryListItemTermProps) {
 	return (
-		<Flex as="dt" fontWeight="bold" width={['100%', '30%']} minWidth="200px">
+		<Flex
+			as="dt"
+			fontWeight="bold"
+			width={['100%', '30%']}
+			minWidth="200px"
+			fontSize="sm"
```

Comment:
Add font size here for `SummaryListItemTerm` and `SummaryListItemDescription` as this will add font/size line height for these elements.